backend/api.test/EndpointTest.cs

```diff
@@ -467,7 +540,7 @@ public async Task SafePositionTest()
                 "application/json"
             );
 
-            await PopulateAreaDb(testInstallation, testPlant, testDeck, testArea);
+            _ = await PopulateAreaDb(testInstallation, testPlant, testDeck, testArea);
```

Comment:
is the _ = really necessary ?

Reply:
I just think it looks nice to acknowledge that we are purposefully ignoring some return variables.